.github/ISSUE_TEMPLATE/1_bug_report.md

```diff
@@ -9,7 +9,6 @@ labels: bug
 ## Environment data
 
 -   VS Code version: XXX
--   Extension version (available under the Extensions sidebar): XXX
```

Comment:
Duplicate entry

Reply:
Question, we are separate, but given our shared scenarios should this entry be changed to:
- Python Extension version (if installed): XXX
and put it below the Jupyter Extension version?

Reply:
Done.